### PR TITLE
Update README with more setup information

### PR DESCRIPTION
**Pull request by nazywam**

### CODE REVIEW

**Review by ITAYC0HEN — APPROVED** [review]

**Comment by ITAYC0HEN** [comment]
Looks good

File: README.md [hunk 1]
```diff
@@ -45,4 +56,10 @@ $ pip install karton-archive-extractor
 $ karton-archive-extractor
 ```
 
+## Running in Docker
+
+Sflock uses [ZipJail](https://github.com/hatching/tracy/tree/master/src/zipjail) as a usermode syscall filtering mechanism. As a result, the karton service has to be able to use `ptrace` so remember to add the `SYS_PTRACE` capability if using Docker.
```
Comment (msm-code) [inline README.md:61]:
```suggestion
Sflock uses [ZipJail](https://github.com/hatching/tracy/tree/master/src/zipjail) as a usermode syscall filtering mechanism. As a result, the karton service has to be able to use `ptrace` so ensure that your container has `SYS_PTRACE` capability.
```

Maybe add example docker invocation :thinking: 

Also I don't think this is true - `SYS_PTRACE` is not necessary to trace your own children. Otherwise it would be impossible to use gdb without root, for example (non-root processes usually have no capabilities at all). Problem is somewhere else, probably :thinking:. Maybe rephrase that as a `debugging step`?

File: README.md [hunk 2]
```diff
@@ -37,6 +37,17 @@ Performs extraction of known archive types and e-mail attachments. Produces "raw
 
 First of all, make sure you have setup the core system: https://github.com/CERT-Polska/karton
 
+In order to unpack all available formats you'll also need a few native dependencies that sflock relies on, so make sure you install them with:
+```shell
+RUN sed -i 's/ main/ main non-free/' /etc/apt/sources.list \
```
Comment (msm-code) [inline README.md:42]: Telling people to run sed on `/etc/apt/sources.list` is a bit shady :thinking:. But I guess technical people will know what it does and adjust accordingly?

**Review by msm-code — CHANGES_REQUESTED** [review]
(actually, request changes instead of a comment)

File: README.md (hunk 2 above)
Comment (nazywam) [reply README.md:42]: This command was copied directly from sflocks readme, seems kinda sketchy though. Should we omit it and just make the user worry about switching to non-free?

Comment (msm-code) [reply README.md:42]: Maybe add `(copied from sflock readme)`. That way, when something breaks it's not technically out fault :duck:

**Review by msm-code — APPROVED** [review]
